lib/node_modules/@stdlib/lapack/base/README.md

```diff
@@ -22,6 +22,161 @@ limitations under the License.
 
+<section class="intro">
+
+## Band Storage
+
+Many LAPACK routines work with banded matrices, which are stored compactly in two-dimensional arrays arranged in linear memory in order to save memory and improve computational efficiency. The following are the different band storage formats used throughout LAPACK.
+
+### General Band Matrix Storage
+
+A general band matrix of size `M`-by-`N` with `KL` subdiagonals and `KU` superdiagonals is stored in a two-dimensional array `AB` with `KL+KU+1` rows and `N` columns.
+
+**Storage Mapping:**
+
+-   Columns of the original matrix are stored in corresponding columns of the array `AB`.
+-   Diagonals of the matrix are stored in rows of the array `AB`.
+-   Element `A[i, j]` from the original matrix is stored in `AB[KU+1+i-j, j]`.
+-   Valid range for `i`: `max(1, j-KU) <= i <= min(M, j+KL)`.
+
+#### Example
+
+Let `M = N = 5`, `KL = 2`, and `KU = 1`. Given an original band matrix `A`
+
+<!-- <equation class="equation" label="eq:band_matrix_a" align="center" raw="A = \left[\begin{array}{rrrrr}a_{11} & a_{12} & 0      & 0      & 0 \\a_{21} & a_{22} & a_{23} & 0      & 0 \\a_{31} & a_{32} & a_{33} & a_{34} & 0 \\0      & a_{42} & a_{43} & a_{44} & a_{45} \\0      & 0      & a_{53} & a_{54} & a_{55}\end{array}\right]" alt="Representation of band matrix A."> -->
+
+```math
+A = \left[
+\begin{array}{rrrrr}
+    a_{11} & a_{12} & 0      & 0      & 0 \\
+    a_{21} & a_{22} & a_{23} & 0      & 0 \\
+    a_{31} & a_{32} & a_{33} & a_{34} & 0 \\
+    0      & a_{42} & a_{43} & a_{44} & a_{45} \\
+    0      & 0      & a_{53} & a_{54} & a_{55}
+\end{array}
+\right]
+```
+
+<!-- </equation> -->
+
+the band storage matrix `AB` is then
+
+<!-- <equation class="equation" label="eq:band_storage_ab" align="center" raw="AB = \left[\begin{array}{rrrrr}*      & a_{12} & a_{23} & a_{34} & a_{45} \\a_{11} & a_{22} & a_{33} & a_{44} & a_{55} \\a_{21} & a_{32} & a_{43} & a_{54} & *      \\a_{31} & a_{42} & a_{53} & *      & *\end{array}\right]" alt="Band storage representation of matrix A."> -->
+
+```math
+AB = \left[
+\begin{array}{rrrrr}
+    *      & a_{12} & a_{23} & a_{34} & a_{45} \\
+    a_{11} & a_{22} & a_{33} & a_{44} & a_{55} \\
+    a_{21} & a_{32} & a_{43} & a_{54} & *      \\
+    a_{31} & a_{42} & a_{53} & *      & *
+\end{array}
+\right]
+```
+
+<!-- </equation> -->
+
+`AB` is a 4×5 matrix as `KL+KU+1 = 2+1+1 = 4`. Elements marked `*` need not be set and are not referenced by LAPACK routines.
+
+**Note:** When a band matrix is supplied for LU factorization, space must be allowed to store an additional `KL` superdiagonals, which are generated by fill-in as a result of row interchanges. This means that the matrix is stored according to the above scheme, but with `KL + KU` superdiagonals.
+
+### Triangular Band Matrices
+
+Triangular band matrices are stored in the same format as general band matrices:
+
+-   If `KL = 0`, the matrix is upper triangular.
```

Comment:
It would be good to actually show matrices which are upper and lower triangular here.